packages/account/src/Components/api-token/__tests__/api-token.spec.js

```diff
@@ -0,0 +1,331 @@
+import React from 'react';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { getPropertyValue, isDesktop, isMobile, useIsMounted } from '@deriv/shared';
+import ApiToken from '../api-token';
+
+const footer_portal_root = document.createElement('div');
+document.body.appendChild(footer_portal_root);
+const overlay_portal_root = document.createElement('div');
+document.body.appendChild(overlay_portal_root);
```

Comment:
@yauheni-kryzhyk-deriv I suggest placing this part in beforeAll() inside describe() 🙂

Reply:
@maryia-binary thanks for the remark) I'll take a note of this :)
done👍